apicast/src/configuration/service.lua

```diff
@@ -137,7 +137,7 @@ function backend_version_credentials.version_oauth(config)
   -- Resource servers MUST support this method. [Bearer]
   access_token = access_token or authorization.token
 
-  local o = oauth.new(self)
+  local o = oauth.new(_M)
```

Comment:
if I use self luacheck complains `self` has not been assigned, so not sure how to do it without changing the method signature, any suggestions?